app/_src/mesh/features/provenance-verification-binaries.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Verify Build Provenance for Kong Mesh Binaries
+badge: enterprise
+---
+
+Starting with 2.8.0, {{site.mesh_product_name}} produces build provenance for binary artifacts, which can be verified using `cosign` / `slsa-verifier` with attestations published to a Docker Hub repository.
+
+This guide provides steps to verify build provenance for signed {{site.mesh_product_name}} binary artifacts using:
+
+* An example, leveraging optional annotations for increased trust
+
+For the example, you will need a compressed binary file and provenance file as well as any of the optional annotations you wish to verify:
+
+| Shorthand | Description | Example Value |
+|---|---|---|
+| `<repo>` | GitHub repository | `kong-mesh` |
+| `<version>` | Artifact version to download | `{{page.version}}` |
+| `<binary-files>` | Compressed binary files for the specified version | `kong-mesh-{{page.version}}-*-*.tar.gz` |
+| `<provenance-file>` | Binary provenance file | `kong-mesh.intoto.jsonl` |
+
+Because Kong uses GitHub Actions to build and release, Kong also uses GitHub's OIDC identity to generate build provenance for binary artifacts, which is why many of these details are GitHub-related.
+
+## Prerequisites
+
+1. Ensure [slsa-verifier](https://github.com/slsa-framework/slsa-verifier?tab=readme-ov-file#installation) is installed.
+
+2. [Download security assets](https://packages.konghq.com/public/kong-mesh-binaries-release/raw/names/security-assets/versions/{{page.version}}/security-assets.tar.gz) for the required version of {{site.mesh_product_name}} binaries
+
+3. Extract the downloaded `security-assets.tar.gz` to access the provenance file `kong-mesh.intoto.jsonl`
+
+   ```sh
+   tar -xvzf security-assets.tar.gz
+   ```
+
+4. [Download compressed binaries](https://cloudsmith.io/~kong/repos/kong-mesh-binaries-release/packages/?q=name%3Akong-mesh-*+version%3A{{page.version}}) for the required version  of {{site.mesh_product_name}}
+
+{:.important .no-icon}
+> The GitHub owner is case-sensitive (`Kong/kong-mesh` vs `kong/kong-mesh`).
+
+## Example
+
+### Using slsa-verifier
+
+```sh
+slsa-verifier verify-artifact \
+   --print-provenance \
+   --provenance-path 'kong-mesh.intoto.jsonl' \
```

Comment:
path doesn't reflect actual security assets should be:

```suggestion
   --provenance-path 'security-assets/kong-mesh.intoto.jsonl' \
```

Reply:
@lahabana  When we extract the `security-assets.tar.gz`, the files are downloaded directly and there is NO parent folder with the name `security-assets`. Do you see / test it something different? Otherwise this seems correct

Reply:
![Screenshot 2024-06-26 at 9 26 14 AM](https://github.com/Kong/docs.konghq.com/assets/29549979/8b48c6a7-ee89-4461-979b-533e5a78b50e)